components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import Link from "next/link"
+import React from "react"
+import styles from "./footer.module.css"
+
+const Footer = () => {
+	return (
+		<div className={styles.footer}>
+			<div className={styles.footer__top}>
+				<ul className={styles.footer__box}>
+					<li className={styles.footer__linkName}>Community</li>
+					<li className={styles.footer__linkItems}>
+						<a
+							target="_blank"
+							rel="noreferrer"
+							href="https://github.com/AccessibleForAll"
+							className={styles.footer__links}>
+							Github Page
+						</a>
+					</li>
+					<li className={styles.footer__linkItems}>
+						<a
+							target="_blank"
+							rel="noreferrer"
+							href="https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/CONTRIBUTING.md"
+							className={styles.footer__links}>
+							{" "}
+							Become a Contributor
+						</a>
+					</li>
+					<li className={styles.footer__linkItems}>
+						<a href="#" className={styles.footer__links}>
+							Sponsor Us
+						</a>
+					</li>
+				</ul>
+				<ul
+					className={styles.footer__box}
+					style={{ paddingInlineStart: "0px" }}>
+					<li className={styles.footer__linkName}>Company</li>
+					<li className={styles.footer__linkItems}>
+						<Link href="/">
+							<a className={styles.footer__links}>Home</a>
+						</Link>
+					</li>
+					<li className={styles.footer__linkItems}>
+						<Link href="/about">
+							<a className={styles.footer__links}>About</a>
+						</Link>
+					</li>
+					<li className={styles.footer__linkItems}>
+						<Link href="/about">
+							<a className={styles.footer__links}>Contact Me</a>
+						</Link>
+					</li>
+				</ul>
+			</div>
+			<div className={styles.footer__bottomDetails}>
+				<span className={styles.footer__copyrightText}>
+					© Accessible Web Dev by{" "}
+					<b className={styles.footer__copyrightTextName}>Accessible For All</b>
+					. Making accessibility accessible for everyone!
+				</span>
+				<div className={styles.footer__mediaIcon}>
+					<a
+						href="https://github.com/AccessibleForAll"
+						target="_blank"
+						rel="noreferrer">
+						<svg
```

Comment:
React icons is installed in the project. Please use an icon from here instead: https://react-icons.github.io/react-icons/search?q=github